bundle/language_en.properties

```diff
@@ -291,6 +291,7 @@ angal.common.new.btn
 angal.common.new.btn.key                                                                               = N
 angal.common.newpatient.btn                                                                            = New Patient
 angal.common.newpatient.btn.key                                                                        = N
+angal.common.nextpage                                                                                  = Next Page
```

Comment:
This key is not currently being used.   If it is then I suspect it will be as a button then it should be `angal.common.nextpage.btn`; similar comment for `angal.common.previouspage` below.

Reply:
You are right, I've opted for symbols "<" and ">", we can remove it now, thanks!